packages/vitest/src/utils/index.ts

```diff
@@ -8,15 +8,18 @@ import type { Suite, Task } from '../types'
 import { EXIT_CODE_RESTART } from '../constants'
 import { getWorkerState } from '../utils'
 import { getNames } from './tasks'
+import { isBrowser, isNode } from './env'
 
 export * from './tasks'
 export * from './base'
 export * from './global'
 export * from './timers'
 
-export const isNode = typeof process < 'u' && typeof process.stdout < 'u' && !process.versions?.deno && !globalThis.window
+export { isNode, isBrowser } from './env'
+// export const isNode = typeof process < 'u' && typeof process.stdout < 'u' && !process.versions?.deno && !globalThis.window
```

Comment:
feel free to remove comments!

Reply:
Upps, I also need  add `export * from ./env`